README.md

```diff
@@ -90,7 +90,138 @@ go run trigger-pipeline/main.go --pipeline-name hello-pipeline --test-image dog.
+        # make a copy of the underlying NumPy array and store it if it is
+        # required.
+        for request in requests:
+            # Perform inference on the request and append it to responses list...
+
+        # You must return a list of pb_utils.InferenceResponse. Length
+        # of this list must match the length of `requests` list.
+        return responses
+
+    def finalize(self):
+        """`finalize` is called only once when the model is being unloaded.
+        Implementing `finalize` function is optional. This function allows
+        the model to perform any necessary clean ups before exit.
+        """
+        print('Cleaning up...')
+```          
+To support 3rd libraries using in Python Backend, Triton build separate Conda environment. The above downloaded Conda environment contain Numpy and scikit-learn library. If you need any other 3rd libraries, you could create your own Conda environment follow [this](https://github.com/triton-inference-server/python_backend#2-packaging-the-conda-environment)
+After creating your own models (AI model and Python Backend pre-processing and post-processing and [ensemble model](https://github.com/triton-inference-server/server/blob/main/docs/architecture.md), please zip all models into zip format before uploading into VDP.
+
+#### Model standardization format output
+When creating a model usingn VDP, you could provide CV Task such as cls (i.e classification), dst (i.e detection). The VDP will format output of model into standarized structure. 
+Without specified cv task, the output will be Triton raw message [ModelInferResponse](https://github.com/kserve/kserve/blob/master/docs/predict-api/v2/grpc_predict_v2.proto) format
+For classification task, the format will be
+```
+{
+    "contents": [
+        {
+            "contents": [
+                {
+                    "category": "dog",
+                    "score": 0.980409,
+                    "box": {
+                        "top": 99.084984,
+                        "left": 325.79257,
+                        "width": 204.18991,
+                        "height": 402.58002
+                    }
+                }
+            ]
+        }
+    ]
+}
+```
+For object detection task, the format will be
+```
+{
+    "contents": [
+        {
+            "category": "dog",
+            "score": 0.998824
+        }
+    ]
+}
+```
+Note: To output with standardized output, the post-processing python model should provide below format
+For classification task, the Python post-processing should output string with format:
+```
+{label name}:{label category}:{score} 
+Example: dog:16:0.998824
+```  
+For object detection task, the Python post-processing should output 2 arrays for label and box objection:
+Label array:
+```
+labels: [[label], label]]
+```
+Box object array:
+```
+bboxes: [[Left, Top, Width, Height, Score], [Left, Top, Width, Height, Score]]
+```
```

Comment:
```suggestion
```
If no CV task is specified when creating a model in VDP, the inference output will be in Triton raw message [ModelInferResponse](https://github.com/kserve/kserve/blob/master/docs/predict-api/v2/grpc_predict_v2.proto) format.
```